src/fastoad/io/configuration/configuration.py

```diff
@@ -249,6 +271,35 @@ def set_optimization_definition(self, optimization_definition: Dict):
         subpart = {"optimization": subpart}
         self._data.update(subpart)
 
+    def _configure_driver(self, prob):
+        driver_config = self._data.get(KEY_DRIVER, {})
+
+        # Check if driver_config is a string (old syntax)
+        if isinstance(driver_config, str):
+            driver_instance_str = driver_config
+            prob.driver = self._om_eval(driver_instance_str)
+        else:
+            # Use new syntax
+            # Set the driver instance
+            driver_instance = driver_config.get("instance")
+            if driver_instance:
+                driver_instance = self._om_eval(driver_instance)
+                prob.driver = driver_instance
+            else:
+                # Raise error if no driver instance is provided
+                raise FASTConfigurationBaseKeyBuildingError(
+                    ValueError("No driver instance provided in configuration file"),
+                    KEY_DRIVER,
+                )
+
+            # Iterate over all keys (attributes) in driver_config except for 'instance'
+            for first_key, first_value in driver_config.items():
+                if first_key != "instance":
+                    # Iterate over all keys (values) in first_value
+                    for second_key, second_value in first_value.items():
+                        # driver.<first_key>[<second_key>] = second_value
+                        getattr(prob.driver, first_key)[second_key] = second_value
+
```

Comment:
This would do the same job:
```suggestion
            for key, value in driver_config.items():
                if key != "instance":
                    getattr(prob.driver, key).update(value)
```